packages/core/helper-plugin/.eslintrc.js

```diff
@@ -1,4 +1,31 @@
 module.exports = {
   root: true,
-  extends: ['custom/front'],
+  extends: ['custom/front/typescript'],
+  parserOptions: {
+    project: ['./tsconfig.eslint.json'],
+  },
+  settings: {
+    'import/resolver': {
```

Comment:
Surprised this isn't picked up via the parserOptions already

Reply:
yeah i was following the set up instructions from here – https://github.com/import-js/eslint-import-resolver-typescript

Reply:
ok, we never needed it for the other ts package I'd be curious to know why someday :D